defaults/main.yml

```diff
@@ -45,6 +45,8 @@ php_upload_max_filesize: "64M"
 php_post_max_size: "32M"
 php_date_timezone: "America/Chicago"
 
+php_allow_url_fopen: "Off"
```

Comment:
Unfortunately, because this defaults to 'On' in PHP currently, and because there are dozens of apps/frameworks that are already using the functionality (instead of curl, Guzzle, or other solutions), I'd rather default this to "On" so I don't break all those apps with the next role version bump.

There _are_ valid use cases for setting this to On, though as phpsec and you point out, it should be pretty rare, and for most people, it's so much safer to _not_ leave it On :)